File: app/src/main/java/maderski/bluetoothautoplaymusic/bus/BusProvider.java
package maderski.bluetoothautoplaymusic.bus;

import com.squareup.otto.Bus;
import com.squareup.otto.ThreadEnforcer;

/**
 * Created by Jason on 11/4/17.
 */

public class BusProvider {
    private static final Bus BUS = new Bus(ThreadEnforcer.MAIN);

    public static Bus getBusInstance() {
        return BUS;
    }

    private BusProvider() {}
}

File: app/src/main/java/maderski/bluetoothautoplaymusic/bus/BusProvider.kt
package maderski.bluetoothautoplaymusic.bus

import com.squareup.otto.Bus
import com.squareup.otto.ThreadEnforcer

/**
 * Created by Jason on 11/4/17.
 */

object BusProvider {
    @JvmStatic
    val busInstance = Bus(ThreadEnforcer.MAIN)
}
